### PR TITLE
Fix failing test while only reporting failure once

### PR DESCRIPTION
**Pull request by lackita**

### CODE REVIEW

File: test/com/gfredericks/test/chuck/clojure_test_test.clj [hunk 1]
```diff
@@ -18,6 +18,24 @@
       (swap! c inc)
       (is (> @c 0)))))
 
+(deftest exception-detection-test
```
Comment (nberger) [inline test/com/gfredericks/test/chuck/clojure_test_test.clj]: This test was moved to [/test/com/gfredericks/test/chuck/exception_handling_test.clj](https://github.com/gfredericks/test.chuck/blob/master/test/com/gfredericks/test/chuck/exception_handling_test.clj) in master. This change should be done in that file, not here.

Comment (lackita) [inline test/com/gfredericks/test/chuck/clojure_test_test.clj]: Ah, I thought I had accidentally deleted the test while I was making my change. Renamed the test in the correct place.

**Comment by nberger** [comment]
Even though this change fixes the exception detection mechanism, I think we are not detecting & reporting failures correctly now, and we need a test to verify it. I created #26 to add that test.

**Comment by gfredericks** [comment]
I agree that there are some improvements to be made, in particular see my comment on #26 about crashing inside vs outside an assertion.

In any case I'm going to merge this because I want the build passing to prevent regressions.

Thanks @lackita!

**Pull request merged**